web/source/osk/visualKeyboard.ts

```diff
@@ -2115,9 +2101,10 @@ namespace com.keyman.osk {
       bs.height=bs.maxHeight=paddedHeight+'px';
 
       b = this.kbdDiv.firstChild as HTMLElement;
+      let gs = this.kbdDiv.style;
       bs=b.style;
       // Sets the layer group to the correct height.
-      bs.height=bs.maxHeight=paddedHeight+'px';
+      gs.height=gs.maxHeight=paddedHeight+'px';
```

Comment:
Part of `VisualKeyboard.adjustHeights()`, which is only called for non-desktop form factors.